cirq/experiments/qubit_characterizations_test.py

```diff
@@ -40,6 +42,37 @@ def test_rabi_oscillations():
     assert rms_err < 0.1
 
 
+def test_single_qubit_cliffords():
```

Comment:
Since one of your goals was to ensure at most one microwave gate in each decomposition, could you add a test for this? Otherwise, this can change on you later unexpectedly.

Reply:
Good idea. Done.